### DISCUSSION TITLE
Are there specific or known reasons to avoid using the updated eddy (from FSL 6.0.7.8/6.0.7.9) with QSIprep?

### DISCUSSION THREAD

**Discussion by psadil** [PennLINC/qsiprep#714] [Q&A] [answered]
As I understand, the [official QSIprep image bundles FSL 6.0.5.1](https://github.com/PennLINC/qsiprep_build/blob/b8418cf85d53eb2e6cc45cd0b0f0e7e5e2f3363d/Dockerfile_FSL#L32-L33). However, [6.0.7.8 introduces what sounds like a set of valuable updates to `eddy`](https://fsl.fmrib.ox.ac.uk/fsl/docs/#/development/history/index?id=_6078-27th-february-2024). I'd appreciate any thoughts on running QSIprep with an updated (fixed) version of eddy.

**Comment by mattcieslak** [comment, marked as answer]
We're planning on updating to the 6.0.7.8 in the upcoming release. These updates seem pretty important
